templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,123 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content item.
+   */
+  Parameters:
+    - "content_item" (associative array) (default: predefined structure): format:
+        {
+          type: 'default', (can be 'default','tile'): Card type
+          description: '', (string) (default: ''): Description of card
+          meta: [], (array) (default: []): Meta's for the Card
+          title: {}, (associative array) (default: {}): Predefined structure compatible with Link component. If Card type is a 'tile', only label property is required.
+          image: '', (string) (default: ''): Url/path to background image (non required if Card type is a 'tile')
+          image_position: 'after' (string) (Can be 'before' or 'after'. Is required only if Icon is specified)
+          tags: [], (array) (default: []): List of tags compatible with EC Tag component structure
+          infos: [], (array) (default: []): format: [
+            {
+              label: '', (string) (default: ''): Label of info
+              icon: {}, (associative array) (default: {}): Predefined structure compatible with EC Icon
+            },
+            ...
+          ]
+          links: [], (array) (default: []): List of links (required if Card type is a 'tile'). Format: [
+            {
+              label: (string) (default: ''): Label of link
+              path: (string) (default: ''): Link url (href attribute)
+            },
+            ...
+          ]
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
```

Comment:
Is this description still relevant? It seems to be related to the `card` component.

Reply:
This description for the object content-item is required to be clear.
All the description is updated.